common/lib/api-client/transformers/move.transformer.js

```diff
@@ -1,6 +1,38 @@
 const { get } = require('lodash')
 // Using get because testcafe barfs on optional chaining
 
+const i18n = require('../../../../config/i18n')
+
+const setMissingToLocation = data => {
+  if (!data.to_location) {
+    const hasOwnLabel = ['prison_recall', 'video_remand']
+
+    data.to_location = {
+      key: `unknown__${data.move_type}`,
+      title: hasOwnLabel.includes(data.move_type)
+        ? i18n.t(`fields::move_type.items.${data.move_type}.label`)
+        : i18n.t('fields::move_type.items.unknown.label'),
+    }
+  }
+}
+
+const setVehicleRegistration = data => {
+  data._vehicleRegistration =
+    get(data, 'meta.vehicle_registration') || undefined
+}
+
+const setExpectedTime = data => {
+  data._expectedCollectionTime =
+    get(data, 'meta.expected_collection_time') || undefined
+  data._expectedArrivalTime =
+    get(data, 'meta.expected_time_of_arrival') || undefined
+}
+
+const setExtraStates = data => {
+  data._hasLeftCustody = ['in_transit', 'completed'].includes(data.status)
```

Comment:
You've separated out `hasOwnLabel` into it's own variable - but you haven't here. Any specific reasoning for that?

Reply:
And, depending upon the answer to the above question, these status here also don't really need to be scoped inside the function, as they're groups of statuses, which are independent of these functions.

i.e. are the `hasLeftCustody` statuses always going to be these two?

Reply:
>You've separated out hasOwnLabel into it's own variable - but you haven't here. Any specific reasoning for that?

Not quite sure what you mean by this?

The reason for creating these properties on each move is to keep them within one layer when we transform a resource from the API layer.

These properties can then be used to do sorting/grouping/filtering without needing to define the set of statuses that make up this "status" in multiple places within the app.

Reply:
Ok, so we have 3 arrays of statuses here:
` const hasOwnLabel = ['prison_recall', 'video_remand']`
`  data._hasLeftCustody = ['in_transit', 'completed'].includes(data.status)`
` data._hasArrived = ['completed'].includes(data.status)`

More clearly my questions were:
1. Why is hasOwnLabel separated out into it's own property, and the other two are inlined?
2. Are these arrays of statuses really only relevant inside each function closure or does the concept of "hasArrived" always have those statuses, in which case, should these values be living in config somewhere?

Reply:
>Why is hasOwnLabel separated out into it's own property, and the other two are inlined?

Ah, gotcha. In short, different developer style/time of writing 😅 the `hasOwnLabel` is actually taken from a previous implementation within the `moves-by-location.js` presenter.

Happy to make these more consistent in this implementation though :) 

>Are these arrays of statuses really only relevant inside each function closure or does the concept of "hasArrived" always have those statuses, in which case, should these values be living in config somewhere?

I would say these "statuses" would always be based on those API statuses. The reason for moving them here is that anytime anything needs to do something with a move based on this value is can know it is consistent rather than having to repeat the status check. If the API introduced another status in the future we would only need to update it here once.

As for config...I don't mind either way. I didn't want to overload the config and this is an isolated enough place to define that property. It's also similar to what we already do with the assessment transformer and person transformer.

Reply:
Cool :)

I'm not opposed to anything in the PR, just pointing out this area of note - I've approved the PR as there's no reason to hold it up for these tweaks